src/components/Cookies/index.jsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react'
+
+import dutchXLogo from 'assets/dutchx.png'
+import ButtonCTA from '../ButtonCTA'
+
+import localForage from 'localforage'
+
+import './_cookies.scss'
+
+class Cookies extends React.Component {
+    state = {
+      settingsOpen: false,
+      cookieSettings: {
+        necessary: true,
+        analytics: false,
+      },
+      loading: false,
+      error: undefined,
+    }
+
+    async componentDidMount() {
+      try {
+        this.setState({ loading: true })
```

Comment:
why `loading: true` here as opposed to in constructor?

Reply:
:+1: